portal/src/main/webapp/js/src/load-frontend.js

```diff
@@ -56,11 +56,21 @@ window.loadReactApp = function(config) {
         });
 
     } else {
+        if (window.frontendUrl === undefined) {
+            if (console.error) {
+                console.error('ERROR: No frontend URL defined, should at least be empty string');
+            } else {
+                console.log('ERROR: No frontend URL defined, should at least be empty string');
+            }
+            // even though this should never happen, probably better to include
```

Comment:
@alisman yeah that's maybe better. Should never happen anyway. It will just try to include a script tag with `undefined` plus you'll get the error message above, so that should be p obvious